centos-7.template

```diff
@@ -49,7 +49,7 @@ nfs-utils
 go-hvkvp
 python-setuptools
 dnsmasq
-python-rhsm-certificates
+subscription-manager-rhsm-certificates
```

Comment:
Any pointer `subscription-manager-rhsm-certificates` is replacing `python-rhsm-certificates`?

Reply:
I think I got what I was looking for in https://git.centos.org/blob/rpms!subscription-manager.git/12d94dd47b3a4d492c5eb9da872dde7e763c3558/SPECS!subscription-manager.spec#L367